lib/errors.js

```diff
@@ -108,6 +103,11 @@ var errors = {
         this.name = 'TARGET_SERVER_ERROR';
         this.message = 'There was an error redirecting the request to the target server: ' + msg;
         this.code = 501;
+    },
+    UserNotFound: function() {
+        this.name = 'USER_NOT_FOUND';
```

Comment:
Looking to documentation, I would have expected "USER_CREDENTIALS_NOT_FOUND". Is USER_NOT_FOUND intentional?